src/main/java/com/zaxxer/nuprocess/internal/LibC.java

```diff
@@ -27,14 +27,23 @@
 @SuppressWarnings("WeakerAccess")
 public class LibC
 {
+    
+   public static int SYS___pthread_chdir = 348; 
```

Comment:
It doesn't seem like this should be mutable. This is clearly intended to be a constant.

Rather than initializing this here and changing it in the `static` block, which requires this to be mutable, leave it uninitialized here and update the `static` block to set the "right" value:
```java
public static final int SYS___pthread_chdir;

static {
   // O_NONBLOCK initialization as now

   SYS___pthread_chdir = System.getProperty("os.name").toLowerCase().contains("freebsd") ? 12 : 348;

   // Register Native
}
```

If this constant is going to move to `LibC`, should it be further down, with the others (like `O_NONBLOCK`)? Alternatively, perhaps the constant should just stay where it was, paired with a new `static` block inside `SyscallLibrary` to initialize it?